### ISSUE TITLE
License

### ISSUE THREAD

**Issue by littledan** [domenic/package-name-maps#47]
I'm wondering, how is this repository licensed? I noticed that there are contributions here from both Googlers and non-Googlers (e.g., @guybedford), so relicensing later might not be trivial. I don't see a license file checked in. For a standards project like this, I imagine that the license would cover both copyright and royalty-free patent policy. Note that [working within a standards body](https://github.com/drufball/layered-apis/issues/31) would solve the license issue.

**Comment by domenic** [comment]
Good catch. The reference implementation is under BSD-3-Clause, as noted in https://github.com/domenic/package-name-maps/blob/master/package-name-map/package.json, but probably it's a good idea to add a LICENSE.txt. The README should be under CC0 or CC-BY most likely.

I think where the work is done is orthogonal; it's easy for an unfortunate maintainer to forget to license their work no matter where their repository is hosted.
